apps/src/applab/designElements/PropertyRow.jsx

```diff
@@ -61,7 +94,8 @@ var PropertyRow = React.createClass({
         type={this.props.isNumber ? 'number' : undefined}
         value={this.state.value}
         onChange={this.handleChangeInternal}
-        style={rowStyle.input} />;
+        onBlur={this.props.isIdRow ? this.onIdRowBlur : null}
```

Comment:
Other option would be to always call this.onBlur and have it exit immediately if !this.props.isIdRow. Probably a six of one half dozen of the other sort of thing.